source/adapters/level_zero/image.cpp

```diff
@@ -747,6 +753,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesSampledImageCreateExp(
                                    pImageDesc, hSampler, phImage));
   return UR_RESULT_SUCCESS;
 }
+} // namespace ur::level_zero
 
 ur_result_t ur_queue_handle_legacy_t_::bindlessImagesImageCopyExp(
     void *pDst, void *pSrc, const ur_image_format_t *pImageFormat,
```

Comment:
Personally, I don't like having namespace twice with this gap in the middle. I'd prefer if the copy function were moved below to allow for one contiguous namespace declaration in the file.

Reply:
I wanted to avoid moving too much stuff, but it seems unavoidable. I'll just add the namespace for everything.